observer/fly.toml

```diff
@@ -7,3 +7,4 @@ primary_region = "cdg"
 
 [deploy]
   strategy = "rolling"
+  release_command = "npm run migrate --workspace observer"
```

Comment:
This will migrate both our DB (spark_stats) and the spark-evaluate DB. That does not seem right to me. WDYT?

https://github.com/filecoin-station/spark-stats/blob/e056ce21f6f355555806d31cd0290a4e5bd45f07/observer/bin/migrate.js#L3-L5

Reply:
You're right! What about we remove `migrateEvaluateDB()` from this bin file?

Reply:
I think that's going to break the second & third use case, won't it?

https://github.com/filecoin-station/spark-stats/pull/140#pullrequestreview-2112812928
> 1. deploying to prod
> 2. running tests
> 3. starting the service locally

How about adding a new `bin/migrate.js` script to the `db` sub-package?

Reply:
good idea!